sdk/program/src/message/sanitized.rs

```diff
@@ -427,33 +427,18 @@ impl SanitizedMessage {
 /// Transaction signature details including the number of transaction signatures
 /// and precompile signatures.
 pub struct TransactionSignatureDetails {
```

Comment:
changed this because i need to create one in `resolved_transaction_view`.
I don't see value in the fields being private - so let me know if I'm missing something there.

Reply:
remind me why `resolved_transaction_view` can't use accessors?

Reply:
it could - it can't construct it 😆
I thought this was simpler than adding a `new`, but lmk if that's what you'd prefer

Reply:
I see. I don't have strong opinion on it, just feel like its a convention that public struct in SDK to keep its members accessible via accessors only. Is some sort of versioning necessary to change publicness of exposed SDK type?

Reply:
I'll just revert - even though its' unlikely anyone consumes this, probably shouldn't break interface.